### ISSUE TITLE
Jupyter Notebook Cell in Dark Theme in WSL not colored

### ISSUE THREAD

**Issue by david-macleod** [microsoft/vscode-jupyter#4899]
Forwarding an issue that was priginally filed in the main vscode repo, but is still an issue

The TLDR; is that the dark theme in the jupyter notebook doesn't distinguish with colour between code cells and output (background)

https://github.com/microsoft/vscode/issues/104863

## Environment data

-   VS Code version: 1.53.2
-   Extension version (available under the Extensions sidebar): ms-python.python
-   OS and version: Windows/WSL/Unix Remote
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.8 (deadsnakes)
-   Value of the `python.languageServer` setting: Pylance

## Expected behaviour

See linked issue https://github.com/microsoft/vscode/issues/104863

## Actual behaviour

See linked issue https://github.com/microsoft/vscode/issues/104863

**Comment by DavidKutu** [comment]
Thanks for bringing it up @david-macleod. Our new Notebook Experience solves this. You can try it now with VS Code Insiders.
https://code.visualstudio.com/insiders/

**Comment by david-macleod** [comment]
Ok thanks!
